Library/Backends/ZendEngine2/Backend.php

```diff
@@ -35,6 +35,9 @@ public function getVariableCodePointer(Variable $variable)
         return '&' . $variable->getName();
     }
 
+    /**
+     * {@inheritdoc}
```

Comment:
> Currently some applications have DocBlocks containing just the {@inheritDoc} inline tag to indicate that their complete contents should be inherited. This usage breaks with the PHPDoc Standard as summaries cannot contain inline tags and inheritance is automatic; you do not need to define a special tag for it.

> However, it does make clear that an element has been explicitly documented (and thus not forgotten). As such we are working to include a new (normal) tag in the PHPDoc Standard @inheritDoc that will serve that purpose.

http://phpdoc.org/docs/latest/guides/inheritance.html